commands/help.rb

```diff
@@ -101,16 +101,16 @@ def self.help(command)
       puts <<~EOT
         Look for package(s).
         Usage: crew search [-v|--verbose] [<pattern> ...]
-        If <pattern> is omitted, all packages will be returned.
+        Both the name and description of packages will be searched.
         If the package color is " + "green".lightgreen + ", it means the package is installed.
         If the package color is " + "red".lightred + ", it means the architecture is not supported.
+        If the package color is " + "blue".lightblue + ", it means the architecture is supported but the package is not installed.
         The <pattern> string can also contain regular expressions.
-        If `-v` or `--verbose` is present, homepage, version and license will be displayed.
+        If `-v` or `--verbose` is present, the homepage, version and license of found packages will be displayed.
         Examples:
-          crew search ^lib".lightblue + " will display all packages that start with `lib`.
-          crew search audio".lightblue + " will display all packages with `audio` in the name.
-          crew search | grep -i audio".lightblue + " will display all packages with `audio` in the name or description.
-          crew search git -v".lightblue + " will display packages with `git` in the name along with homepage, version and license.
+          crew search ^lib".lightblue + " will display all packages with a name or description that starts with `lib`.
+          crew search audio".lightblue + " will display all packages with `audio` in the name or description.
```

Comment:
This is not true.   Compare the output of the commands below:
```
$ crew search audio
libaudiofile: The Audio File Library is a C-based library for reading and writing audio files in many common formats.
libfaudio: FAudio is an XAudio reimplementation that focuses solely on developing fully accurate DirectX Audio runtime libraries for the FNA project.
libspatialaudio: libspatialaudio is a library for Ambisonic encoding and decoding, filtering and binaural rendering.
pcaudiolib: PCaudiolib is a portable C audio library.
pulseaudio: PulseAudio is a sound system for POSIX OSes, meaning that it is a proxy for your sound applications.
rtaudio: RtAudio is a set of C++ classes that provide a common API (Application Programming Interface) for realtime audio input/output
webrtc_audio_processing: AudioProcessing library based on Googles implementation of WebRTC

$ crew search | grep -i audio | head -20
aalib: AA means Ascii Art - the AAlib (ascii art GFX library), BB (audiovisual demonstration for your terminal), aview (image browser/animation player), AAvga (SVGAlib wrapper for AA-lib), ttyquake (text mode quake), aa3d (random dot stereogram generator)...
alsa_lib: The Advanced Linux Sound Architecture (ALSA) provides audio and MIDI functionality to the Linux operating system.
audacious: Audacious is an open source audio player.
audacious_plugins: Audacious is an open source audio player.
audacity: Audacity is the world's most popular audio editing and recording app
cdparanoia: Compact Disc Digital Audio extraction tool
cdrkit: cdrkit is a suite of programs for recording CDs and DVDs, blanking CD-RW media, creating ISO-9660 filesystem images, extracting audio CD data, and more.
chromaprint: Chromaprint is a client-side library that implements a custom algorithm for extracting fingerprints from any audio source.
cras: ChromeOS Audio Server
ecasound: Ecasound is a software package designed for multitrack audio processing.
faac: Freeware Advanced Audio Coder
faad2: Freeware Advanced Audio (AAC) Decoder
ffmpeg: Complete solution to record, convert and stream audio and video
flac: FLAC stands for Free Lossless Audio Codec, an audio format similar to MP3, but lossless, meaning that audio is compressed in FLAC without any loss in quality.
gst_editing_services: GStreamer library for creating audio/video editors
jack: JACK (JACK Audio Connection Kit) refers to an API that provides a basic infrastructure for audio applications to communicate with each other and with audio hardware.
jack1: JACK (JACK Audio Connection Kit) refers to an API that provides a basic infrastructure for audio applications to communicate with each other and with audio hardware.
ladspa: Linux Audio Developer's Simple Plugin API
libaudiofile: The Audio File Library is a C-based library for reading and writing audio files in many common formats.
libavc1394: libavc1394 is a programming interface for the 1394 Trade Association AV/C (Audio/Video Control) Digital Interface Command Set.
```

Reply:
You are not running these commands on the updated branch. The behaviour has changed.

Reply:
Oh wait, you changed the search algorithm.  Let me test this.

Reply:
Yeah, you were right.  I didn't update using that branch.  This works good.